zipline/test_algorithms.py

```diff
@@ -894,3 +894,36 @@ def handle_data(context, data):
     context.incr += 1
     record(data=float('%s'))
 """
+
+call_with_kwargs = """
```

Comment:
For completion, we could also test `call_without_kwargs`.
e.g. `data.history(symbol('TEST'), 'price', 5, '1d')`